code/modules/assembly/signaler.dm

```diff
@@ -45,6 +58,7 @@ GLOBAL_LIST_EMPTY(remote_signalers)
 
 /obj/item/assembly/signaler/proc/signal_callback()
 	pulse(1)
+	to_chat(usr, "You have sent the signal.")
```

Comment:
Can you make this a `user`? Also, probably some span notice class, and maybe some rewording
`You activate [src]`?